packages/tooling/fast-tooling/src/web-components/color-picker/color-picker.ts

```diff
@@ -7,11 +7,16 @@ import {
 } from "@microsoft/fast-colors";
 import { attr, DOM, observable } from "@microsoft/fast-element";
 import { isNullOrWhiteSpace } from "@microsoft/fast-web-utilities";
-import { FASTTextField } from "@microsoft/fast-components";
 import { FormAssociatedColorPicker } from "./color-picker.form-associated";
 
-// Prevent tree shaking
-FASTTextField;
+/**
+ * This is currently experimental, any use of the color picker must include the following
+ * imports and register with the DesignSystem
+ *
+ * import { fastTextField } from "@microsoft/fast-components";
+ * import { DesignSystem } from "@microsoft/fast-foundation";
+ * DesignSystem.getOrCreate().register(fastTextField());
```

Comment:
I think you *also* want to change this component to follow the model set in fast-foundation, where ColorPicker extends FoundationElement  the package export is actually the product of `ColorPicker.compose()`.

This gives ColorPicker oppertunity to get the *tag* for any dependent components from context:

```
const ColorPickerTemplate = (context: ElementDefinitionContext) => {
   return html` .... <${context.tagFor(TextField}`></${context.tagFor(TextField)}>`
}

Reply:
This is passing build, I need to also convert the other web components such as the html renderer to use the updated format. I think it might be more timely to merge this as-is with a follow up PR.

Reply:
cool sounds good